test/LogMessageTest.h

```diff
@@ -8,7 +8,7 @@ class LoggerMock : public Logger {
 	LoggerMock();
 	virtual ~LoggerMock();
 
-	MOCK_METHOD2(Log, void(const std::string& message, const std::string& moduleName));
+	MOCK_METHOD1(Log, void(const std::string& message));
```

Comment:
I just realized the number at the end of MOCK_METHOD is the number of arguments a function is supposed to take.

Reply:
Yes. That was also the stumbling block I hit when I first tried writing a few Google Mock tests. The newer versions of Google Mock don't need that. Instead they separate out the return type from the argument types, and are somehow able to detect the number of parameters. The newer code samples use an updated macro.